lib/postprocessing/image.js

```diff
@@ -72,8 +72,6 @@ const EXTRA_INSTRUCTIONS = new Set([
     "watermark"
 ]);
 
-const CHANGE_ORIENTATION_LIST = [ "LeftTop", "RightTop", "RightBottom",  "LeftBottom"];
```

Comment:
With the `-n` option, exiftool returns the numerical value. So it becomes a simpler comparison.